tests/test-image.php

```diff
@@ -88,7 +88,7 @@ public function test_fudge_factor() {
 		$before = (int) ini_get( 'memory_limit' );
 		$format = 'png';
 		$file = __DIR__ . '/data/pb.png';
-		@\Pressbooks\Image\fudge_factor( $format, $file );
+		@\Pressbooks\Image\fudge_factor( $format, $file, 100 ); //adjust fudge to png during tests to avoid false positives
```

Comment:
It seems the garbage collector is doing a great job saving memory so that test was not working because it was enough memory to process the image so I adjusted the resize factor during tests to require more memory and increase it accordingly